### ISSUE TITLE
@apply alternative / abandoned spec

### ISSUE THREAD

**Issue by Vorror** [tailwindcss/tailwindcss#627]
With `@apply` being an [abandoned](http://tabatkins.github.io/specs/css-apply-rule/) specification it's unlikely any syntax parsers, linters, formatters, libraries, etc will support it going forward. 

Will tailwind be moving to something else? Maybe something that also works with scss?

**Comment by adamwathan** [comment]
We won't be switching it unless the spec comes back and makes our version make no sense 👍🏻 We use a lot of other at-rules that aren't real (`@screen`, `@tailwind`, `@variants`, `@responsive`) so even if we changed this one you'd still have the same syntax/linter issues for those. Hopefully one day we can invest in creating our own syntax files for popular editors, but in the mean time I always recommend people use PostCSS syntax highlighting and it seems to work fine.

> Maybe something that also works with scss?

Not sure what you mean here as it already works fine with SCSS. Any particular problem you are hitting?

Closing as not an issue but happy to continue the discussion.
